### PR TITLE
Add e2e test for executing kubectl command on forbidden verbs and resources

### LINKED ISSUE #350: Add e2e test for executing kubectl command on forbidden verbs and resources
**Is your feature request related to a problem? Please describe.**

Missing kubectl command test coverage for negative test-cases

**Describe the solution you'd like**

BotKube can be configured to allow access to specific kubectl commands only - https://github.com/infracloudio/botkube/blob/develop/test/resource_config.yaml#L224

Add a test case to execute kubectl command on a forbidden resource and a verb.
Add the test cases in the table-driven test map https://github.com/infracloudio/botkube/blob/develop/test/e2e/command/kubectl.go#L46

### PR DESCRIPTION
**Pull request by anoopmsivadas**
##### ISSUE TYPE

 - Bug fix Pull Request

##### SUMMARY

Added a e2e test kubectl command with forbidden verb and forbidden resource. Another one for kubctl command with allowed verb and forbidden resource

Fixes #350

### CODE REVIEW

File: test/e2e/command/botkube.go [hunk 1]
```diff
@@ -83,6 +83,10 @@ func (c *context) testBotkubeCommand(t *testing.T) {
 				"  - statefulsets\n" +
 				"  - storageclasses\n",
 		},
+		"BotKube notifier showconfig": {
+			command:  "notifier showconfig",
+			expected: "<@U023BECGF> notifier showconfig",
+		},
```
Comment (PrasadG193) [inline test/e2e/command/botkube.go:89]: @anoopmsivadas Can you please remove this part. This is already being addressed in the separate PR.
The rest looks good to me

Comment (anoopmsivadas) [reply test/e2e/command/botkube.go:89]: @PrasadG193 so I have to remove this part and create a new pull request, right?

**Comment by anoopmsivadas** [comment]
I'm closing this PR.  will try not to make same mistakes with git
